cookbooks/aws-parallelcluster-slurm/templates/default/slurm/cgroup.conf.erb

```diff
@@ -4,3 +4,8 @@
 CgroupAutomount=yes
 TaskAffinity=no
 ConstrainCores=yes
+
+# WARNING!!! The slurm_parallelcluster_cgroup.conf file included
+# gets updated by the pcluster process, be careful
```

Comment:
What do you mean with "be careful"? I suggest to be more specific here, e.g. do not manually change the file, do not manually change some specific properties, else....

Reply:
I took the exact same message from the [template for `slurm.conf`](https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-slurm/templates/default/slurm/slurm.conf.erb#L87) (with slight improvements).

Should we edit all of them (including the one for `gres.conf`) and make them consistent?

Reply:
Discussed with @gmarciani yesterday. We decided to leave the messages as they are.